clusters/data_node_type.go

```diff
@@ -89,6 +94,19 @@ type NodeType struct {
 	Graviton              bool                          `json:"is_graviton,omitempty"`
 }
 
+func (nt NodeType) shouldBeSkipped() bool {
+	shouldBeSkipped := false
+	if nt.NodeInfo != nil {
```

Comment:
i'll quote my favorite chapter from https://livebook.manning.com/book/100-go-mistakes-how-to-avoid-them/chapter-2/v-10/30
![image](https://user-images.githubusercontent.com/259697/185087127-aea682d4-2995-4744-9cc9-adc89fb08b35.png)